providers/nodeReleasesDataProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import { createContext } from 'react';
+import type { FC, PropsWithChildren } from 'react';
+import type {
+  NodeReleaseData,
+  NodeReleasesDataContext as NodeReleasesDataContextType,
+} from '../types';
+
+type NodeReleasesDataProviderProps = PropsWithChildren<{
+  nodeReleasesData: NodeReleaseData[];
+}>;
+
+export const NodeReleasesDataContext =
+  createContext<NodeReleasesDataContextType>(undefined as any);
+
+export const NodeReleasesDataProvider: FC<NodeReleasesDataProviderProps> = ({
+  children,
+  nodeReleasesData,
+}) => {
+  const current = nodeReleasesData.find(
+    nodeReleaseData => nodeReleaseData.status === 'Current'
+  );
+  const lts =
+    nodeReleasesData.find(
+      nodeReleaseData => nodeReleaseData.status === 'Active LTS'
+    ) || current;
+
+  return (
+    <NodeReleasesDataContext.Provider
+      value={{ nodeReleasesData, lts, current }}
```

Comment:
```suggestion
      value={{ releases, lts, current }}
```

I feel the array containing all objects should just be named `allReleaes` or `releases` or `all`, since it's already a given that this context is about node releases data